pilot/pkg/model/authorization.go

```diff
@@ -38,12 +40,34 @@ type RolesAndBindings struct {
 	RoleNameToBindings map[string][]*rbacproto.ServiceRoleBinding
 }
 
+// AuthorizationPolicyConfig stores the AuthorizationPolicy and its name.
+type AuthorizationPolicyConfig struct {
+	Name   string
+	Policy *rbacproto.AuthorizationPolicy
+}
+
+func (cfg AuthorizationPolicyConfig) String() string {
+	return fmt.Sprintf("AuthorizationPolicyConfig{name: %s, policy: %s", cfg.Name, cfg.Policy)
+}
+
+// AuthorizationConfigV2 stores a list of AuthorizationPolicyConfig and ServiceRole in a given namespace.
+type AuthorizationConfigV2 struct {
+	// A list of AuthorizationPolicy.
+	AuthPolicies []*AuthorizationPolicyConfig
```

Comment:
AuthzPolicies (or AuthZPolicies) instead of AuthPolicies?

Reply:
Done.